webapp/store/logic.py

```diff
@@ -248,3 +248,27 @@ def has_stable(channel_maps_list):
                         return True
 
     return False
+
+
+def get_lowest_available_risk(channel_map, default_track):
```

Comment:
We are adding these logic utility functions - it would be nice to have some basic unit tests for them to make sure that give expected results in different cases.

Reply:
Added!